src/dataExplorer/components/FluxQueryBuilder.tsx

```diff
@@ -45,6 +50,10 @@ const FluxQueryBuilder: FC = () => {
     setSelection({bucket: null, measurement: ''})
   }, [setQuery, setStatus, setResult, setSelection, cancel])
 
+  const handleShowOverlay = () => {
+    history.push(`${pathname}/save`)
```

Comment:
using pathname instead of declaring it from selector state is gross and makes a weird coupling between router state and internal state. but its a pattern used elsewhere and i think this is temporary, so whateves.

Reply:
Again, not too invested in anything changing here, but it's that org I'd part that should look to the "currently selected org" single source of truth in the reducer for that info, not the route